app/src/main/res/values/colors.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="primary">#66a966</color>
-    <color name="primaryDark">#4a794a</color>
+    <color name="primary">#7fae09</color>
+    <color name="primaryDark">#476105</color>
```

Comment:
This colour has been created by reducing the 'Brightness' component of the Primary colour from 68 to 38